nuxeo-core/nuxeo-core-mongodb/src/test/java/org/nuxeo/ecm/core/mongodb/TestMongoDBIndices.java

```diff
@@ -83,11 +82,9 @@ public void shouldFailWhenCreatingExistingChildNameDocument() {
             fail("should throw a ConcurrentUpdateException");
         } catch (ConcurrentUpdateException cue) {
             assertEquals(SC_CONFLICT, cue.getStatusCode());
-            assertTrue(
-                    cue.getMessage()
-                       .contains(String.format(
-                               "E11000 duplicate key error collection: %s.test index: ecm:parentId_1_ecm:name_1 dup key: { : \"%s\", : \"%s\" }",
-                               DEFAULT_MONGODB_DBNAME, folder.getId(), DOCUMENT_NAME)));
+            assertTrue(cue.getMessage().contains("E11000 duplicate key error collection"));
+            assertTrue(cue.getMessage().contains(String.format("%s", DOCUMENT_NAME)));
+            assertTrue(cue.getMessage().contains(String.format("%s", folder.getId())));
```

Comment:
Just `contains(DOCUMENT_NAME)` and `contains(folder.getId())`
Also you could put `cue.getMessage()` in a local variable.

Reply:
sorry i didn't know i did with format :(